contributors/devel/issues.md

```diff
@@ -53,11 +150,11 @@ bug shouldn't be in this milestone for more than a day or two towards the end of
 a milestone. It should be triaged either into vX.Y, or moved out of the release
 milestones.
 
-The above priority scheme still applies. P0 and P1 issues are work we feel must
-get done before release.  P2 and P3 issues are work we would merge into the
-release if it gets done, but we wouldn't block the release on it. A few days
-before release, we will probably move all P2 and P3 bugs out of that milestone
-in bulk.
+The above [priority](#define-priority) scheme still applies. P0 and P1 issues
```

Comment:
The numerical priorities (P0, P1, etc.) should be changed to the verbose ones, but that could be done in a subsequent PR.